### PR TITLE
Corrections sujet

### PR DESCRIPTION
**Pull request by Axthauvin**
Corrections d'erreurs !

### CODE REVIEW

**Review by kolowy — CHANGES_REQUESTED** [review]
I think you need to rebase your branch :P

**Comment by kolowy** [comment]
> I think you need to rebase your branch :P

done !
Please review before merge @Axthauvin

File: layouts/shortcodes/interfaces.html [hunk 1]
```diff
@@ -591,7 +591,7 @@ <h1>Guerrier</h1>
     
     <div class="card-body">
       <p>Cette première partie sera surtout consacrée à la création de l'interface et de ses sous-classes. 
-        <br> Si vous ne souhaitez pas faire tout le tp, je vous conseille de faire au moins cette partie et de lire le cours du dessus pour bien sasir le concep.
+        <br> Si vous ne souhaitez pas faire tout le tp, je vous conseille de faire au moins cette partie et de lire le cours du dessus pour bien sasir le concept.
```
Comment (Taliayaya) [inline layouts/shortcodes/interfaces.html:594]: pour bien **saisir** ~~sasir~~ le concept

**Review by Taliayaya — COMMENTED** [review]
lgtm

File: layouts/shortcodes/interfaces.html (hunk 1 above)
Comment (kolowy) [reply layouts/shortcodes/interfaces.html:594]:
```suggestion
        <br> Si vous ne souhaitez pas faire tout le tp, je vous conseille de faire au moins cette partie et de lire le cours du dessus pour bien saisir le concept.
```

**Pull request merged**